drivers/flash/flash_shell.c

```diff
@@ -663,7 +663,7 @@ static int cmd_load(const struct shell *sh, size_t argc, char *argv[])
 	flash_load_boff = 0;
 	flash_load_chunk = 0;
 
-	shell_print(sh, "Loading %d bytes starting at address %x", size, addr);
+	shell_print(sh, "Enter %d characters to load %d bytes starting at address %x", size, size, addr);
```

Comment:
What about: `Waiting for %d bytes to load at address %x`.

Reply:
This sounds better. Maybe "store at address"?

Reply:
Since the command is `flash load` I would keep the "load".

Reply:
> Since the command is `flash load` it seems better to keep the "load".

Yuck, but can not argue with above sentence.

Reply:
@hlord2000, I was under the impression that this would be mainly used to start a separate process to upload flash data, not to type them (although this is also possible). This is why I used the terminology "send".